cjwkernel/types.py

```diff
@@ -615,6 +615,21 @@ def TODO_i18n(cls, text: str) -> I18nMessage:
         """
         return cls("TODO_i18n", {"text": text})
 
+    @classmethod
+    def trans(
+        cls,
+        message_id: str,
+        *,
+        default: str,
+        parameters: Dict[str, Union[int, float, str]] = {},
```

Comment:
Let's call this `args`, so it corresponds with `I18nMesage.args`.

From briefly grepping, I surmise that the `parameters` argument in `cjworkbench.i18n.trans()` is ignored by the extraction logic. it _really_ could be called "args" or "arguments" everywhere. (ICU MessageFormat docs use the term, "arguments".) But I'm not compelled to change it everywhere, because then we'll have the same hassle with "args" here vs "arguments" elsewhere.